crates/api_common/src/request.rs

```diff
@@ -295,32 +302,31 @@ async fn generate_pictrs_thumbnail(image_url: &Url, context: &LemmyContext) -> L
     encode(image_url.as_str())
   );
 
-  let response = context
+  let res: PictrsResponse = context
```

Comment:
Is it no longer able to infer the type?

Reply:
It needs them. You can either add it there, or to the json response below. I'll move it down since that seems to be preferred.